File: Warga.java
package model;

public class Warga {
    private String nik;
    private String nama;
    private String jenisKelamin;
    private String statusHubungan; // Contoh: Kepala Keluarga, Istri, Anak

    public Warga(String nik, String nama, String jenisKelamin, String statusHubungan) {
        this.nik = nik;
        this.nama = nama;
        this.jenisKelamin = jenisKelamin;
        this.statusHubungan = statusHubungan;
    }

    // Getter dan Setter
    public String getNik() { return nik; }
    public void setNik(String nik) { this.nik = nik; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public String getJenisKelamin() { return jenisKelamin; }
    public void setJenisKelamin(String jenisKelamin) { this.jenisKelamin = jenisKelamin; }

    public String getStatusHubungan() { return statusHubungan; }
    public void setStatusHubungan(String statusHubungan) { this.statusHubungan = statusHubungan; }
}

File: warga.py
class Warga:
    def __init__(self, nik: str, nama: str, jenis_kelamin: str, status_keluarga: str):
        self.nik = nik
        self.nama = nama
        self.jenis_kelamin = jenis_kelamin
        self.status_keluarga = status_keluarga  # Contoh: Kepala Keluarga, Istri, Anak

    def to_dict(self):
        """Mengubah data objek menjadi dictionary agar mudah dibaca/dikelola"""
        return {
            "NIK": self.nik,
            "Nama": self.nama,
            "Jenis Kelamin": self.jenis_kelamin,
            "Status": self.status_keluarga
        }
